test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -182,6 +189,20 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | Should -Be 1
     }
 
+
+    It "Test that a hex named script executed without prefix will run case: <case>" -TestCases @(
+            @{ case = "0x.ps1" }
+            @{ case = "-0x.ps1" }
+            @{ case = "+0x.ps1" }
+    ){
+        param ($case)
+        Set-Location $TestDrive
+        New-Item -ItemType File -Name $case -Path $testfolder1 -Value {Write-Output "Hello"}
+        & $case | Should -BeExactly "Hello"
+        $pathtodelete = Join-Path -Path $testfolder1 -ChildPath $case
+        Remove-Item $Pathtodelete -Force
```

Comment:
Since we're adding $testfolder1 to the PATH for pwsh to find anyway, maybe we can remove the Set-Location from this test? Wondering if that's what's causing other tests to fail somehow 🤔